packages/api/src/routes/medical/internal-hie.ts

```diff
@@ -74,7 +74,7 @@ router.post(
     }).catch(err => {
       const msg = `Error unlinking patient from organization`;
       log(`Error unlinking patient from organization: ${errorToString(err)}`);
-      capture.error(err, { extra: { msg, cxId, patientId, oid } });
+      capture.error(msg, { extra: { err, cxId, patientId, oid } });
```

Comment:
nit: pass error as the last parameter, please. The reason is that it might be very large object, and Sentry has a limit on the payload size, so it will discard the excess data... so it's better to lose part of the error message when too large than the cxId, patientId, and oid. Also, while there, please rename it to `error`, that's the name of the property we'll see on Sentry.